src/coreclr/nativeaot/System.Private.CoreLib/src/System/Exception.NativeAot.cs

```diff
@@ -112,6 +117,10 @@ private static void AppendExceptionStackFrame(object exceptionObj, IntPtr IP, in
                 bool isFirstFrame = (flags & (int)RhEHFrameType.RH_EH_FIRST_FRAME) != 0;
                 bool isFirstRethrowFrame = (flags & (int)RhEHFrameType.RH_EH_FIRST_RETHROW_FRAME) != 0;
 
+                // track count for metrics
+                if (isFirstFrame && !isFirstRethrowFrame)
+                    Interlocked.Increment(ref s_exceptionCount);
```

Comment:
I'm going to assume this matches how CoreCLR counts these wrt rethrow or `ExceptionDispatchInfo.Throw`.